ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -443,6 +627,17 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-ipi-zones-multisubnets-external-lb-f28
```

Comment:
Please remove `vsphere-ipi-zones-f28`

Reply:
ignore this comment

Reply:
added the job vsphere-ipi-zones-f28 back

Reply:
added vsphere-ipi-zones-f28 back